src/drc/datamodel/models.py

```diff
@@ -23,8 +25,10 @@
 
 class InformatieObject(models.Model):
     identificatie = models.CharField(
-        max_length=40, validators=[alphanumeric_excluding_diacritic],
-        default=_uuid.uuid4,
+        max_length=40,
+        validators=[alphanumeric_excluding_diacritic],
+        blank=True,
+        default="",
```

Comment:
Ik begrijp waarom blank en default zo zijn maar dan moet het in de serializer verplicht worden want het is een verplicht veld. Moet dus in retro ook zo in zrc.

Reply:
Het is toch net zo dat er een identificatie gegenereerd wordt als je hem weglaat, dus dan kan je het niet verplicht maken in serializer?

Reply:
oh, je hebt gelijk. stom van mij.